### PR TITLE
Re-add low_latency_mode parameter to Servo

### PR DESCRIPTION
**Pull request by sea-bass**
This PR re-adds the `low_latency_mode` workaround to servo, but also fixes a bug I found in doing this where the `new_X_msg_` flags were not being unset correctly.

### CODE REVIEW

**Comment by sea-bass** [comment]
@ibrahiminfinite

File: moveit_ros/moveit_servo/src/servo_node.cpp [hunk 1]
```diff
@@ -202,10 +202,6 @@ void ServoNode::poseCallback(const geometry_msgs::msg::PoseStamped::ConstSharedP
 
 std::optional<KinematicState> ServoNode::processJointJogCommand()
 {
-  std::optional<KinematicState> next_joint_state = std::nullopt;
```
Comment (sea-bass) [inline moveit_ros/moveit_servo/src/servo_node.cpp:205]: woops, re-add this

Comment (sea-bass) [reply moveit_ros/moveit_servo/src/servo_node.cpp:205]:
```suggestion
  std::optional<KinematicState> next_joint_state = std::nullopt;

**Comment by ibrahiminfinite** [comment]
How about we give https://github.com/ros-planning/moveit2/issues/1857#issuecomment-1506581979 a try ?
The `publish_period` no longer makes sense if we add `low_latency_mode`.

**Comment by sea-bass** [comment]
> How about we give [#1857 (comment)](https://github.com/ros-planning/moveit2/issues/1857#issuecomment-1506581979) a try ? The `publish_period` no longer makes sense if we add `low_latency_mode`.

I actually just tried this and it performs almost as well as re-adding this parameter! I'll put up a PR soon

**Pull request closed without merging**